converter/shieldhit/beam.py

```diff
@@ -26,6 +60,9 @@ class BeamConfig:
     beampos: tuple[float, float, float] = (0, 0, 0)  # [cm]
     beamdir: tuple[float, float, float] = (0, 0, 1)  # [cm]
     delta_e: float = 0.03  # [a.u.]
+    nuclear_reactions: bool = True
+    straggle: StraggleModel = StraggleModel.VAVILOV
```

Comment:
```suggestion
    straggle: StraggleModel = StraggleModel.VAVILOV
```

rename to straggling consistently